build.rs

```diff
@@ -53,14 +77,6 @@ fn main() -> Result<(), Box<dyn Error>> {
 
     println!("cargo:rerun-if-changed=conf.pri");
     println!("cargo:rerun-if-changed=src");
-    println!(
```

Comment:
why are we removing these?

Reply:
Ah, their presence causes `cargo build` to want to rebuild again, because those libs are now created after cargo starts running rather than before. In any case they shouldn't be needed since any change in src is sufficient to ensure a rebuild.